integration-test/src/test/java/com/datastrato/gravitino/integration/test/util/spark/SparkTableInfo.java

```diff
@@ -66,6 +76,26 @@ void setBucket(Transform bucket) {
     this.bucket = bucket;
   }
 
+  void addHourPartition(Transform hourPartition) {
```

Comment:
could you use one shared partition list for all partitions,  not the seperate xxxPartitions for the specify partition?

Reply:
> could you use one shared partition list for all partitions, not the seperate xxxPartitions for the specify partition?

Because the partition transforms are finite, this way seems clearer and helps to understand. WDYT?

Reply:
suppose partition expressions is (columnA, year(columnB)),   but Gravitino returns (year(columnB), columnA), it will pass the check unexpectedly.

Reply:
instead of testing each partition function seperately, could we test the combined cases?

Reply:
got it

Reply:
updated.